debian/clearwater-infrastructure.init.d

```diff
@@ -79,6 +79,15 @@ do_start()
         # Reload monit to pick up any changes to its config files
         pkill -HUP monit
 
+        # Restart the socket factories if they're running to pick up any changes
+        # in namespace configuration
+        if ( status clearwater-socket-factory-mgmt 2>&1 | grep start ); then
+          restart clearwater-socket-factory-mgmt
+        fi
+        if ( status clearwater-socket-factory-sig 2>&1 | grep start ); then
+          restart clearwater-socket-factory-sig
+        fi
```

Comment:
This is not strictly necessary as socket factory restarts itself when it is asked for something not on the whitelist. However Steve was concerned that if one factory restarted and not the other it would look like one of them was using the old config. Given that restarting the processes is non-trivial (due to OS differences) I suggest we just remove this code.